pom.xml

```diff
@@ -9,14 +9,15 @@
       <java11.build.outputDirectory>${project.build.directory}/classes-java11</java11.build.outputDirectory>
       <artifact.classifier/>
 
-      <docker.maven.plugin.fabric8.version>0.36.0</docker.maven.plugin.fabric8.version>
+      <docker.maven.plugin.fabric8.version>0.40.2</docker.maven.plugin.fabric8.version>
       <felix.bundle.plugin.version>5.1.1</felix.bundle.plugin.version>
-      <felix.version>6.0.1</felix.version>
+      <felix.version>7.0.5</felix.version>
       <hibernate.version>5.4.24.Final</hibernate.version>
       <javassist.version>3.27.0-GA</javassist.version>
       <jndi.version>0.11.4.1</jndi.version>
       <maven.release.version>2.5.3</maven.release.version>
       <metrics.version>3.2.5</metrics.version>
+      <metrics5.version>5.0.0-rc11</metrics5.version>
```

Comment:
You might naturally question managing this dependency version on an RC, but [they've been RC'ing this for over two years now][1], so I _think_ this is appropriate.

[1]: https://search.maven.org/artifact/io.dropwizard.metrics5/metrics-core

Reply:
This is _still_ the case as of June 2023.  I advanced the RC version to 17 as of a51f146a87ca5ac9702a87b0c79b716b3429c226.